boards/laird_connect/bl654_dvk_bl654pa/Kconfig

```diff
@@ -0,0 +1,10 @@
+# BL654 DVK board configuration
+
+# Copyright (c) 2019 Laird Connectivity
```

Comment:
with hwmv2 this isn't needed, instead it can be added as a variant to `bl654_dvk` so that there is `bl654_dvk` and `bl654_dvk//pa` (or similar), that way it can reuse the base bl654_dvk files then have additional overlay files on top

Reply:
I have updated the patch so this support is a variant of the bl654_dvk now.